### PR TITLE
add a Try it button on the homepage

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,13 +27,23 @@
...
+        <h2 class="text-center">
+          The language-agnostic parts of IPython are getting a new home in Project Jupyter
+        </h2>
+      </div>
     </div>
+    <hr>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2 text-center">
+        <p>
+        <a type="button" class="btn btn-primary btn-lg" href='https://try.jupyter.org'>Try it In your browser</a>
```
Comment (rgbkrk) [inline index.html]: s/In/in/

File: index.html [hunk 2]
```diff
@@ -27,13 +27,23 @@
...
+        </p>
+        <p class=''>No install or login required, with Python 3, Julia and R</p>
```
Comment (rgbkrk) [inline index.html]: What's the empty class for?

I'd make this text read: "No install or login required. Python 3, Julia, and R."

**Comment by Carreau** [comment]
Sorry forgot to push. 
fixed both.

**Comment by rgbkrk** [comment]
My only fear is that we'll get overwhelmed by drive by demand. Only one way to find out!

If it gets to be too much, we can revert this button, revisit our scaling, and rethink how we're doing this. Let's :shipit:

**Comment by rgbkrk** [comment]
Looking good, we're sitting at about 91% available right now.

**Comment by Carreau** [comment]
The peek of tweets is gone. 
Was a quick Buzz. 
maybe this week-end, or when europe wakes up.

**Pull request merged**